tests/unit/test_normalizer.py

```diff
@@ -29,13 +34,12 @@ def test_initialization():
     assert norm.target_channel_axis == 5
 
 
-def test_crafting_image():
-    doc = Document(uri=os.path.join(directory, 'test_image.png'))
+def test_crafting_image(root_dir):
+    doc = Document(uri=os.path.join(root_dir, 'data', 'test_image.png'))
```

Comment:
In tests in Jina we in general do it like the following (and regardless if it is good or bad, I'd like to keep it consistent):
```
# somewhere at the top of the test-file
cur_dir = os.path.dirname(os.path.abspath(__file__))
...
def mytest():
    ... os.path.join(cur_dir, '../data/test_image.png')
```

Reply:
Sure - it is changed